### PR TITLE
Update uploads.py

### PR DESCRIPTION
**Pull request by kesha1225**

### CODE REVIEW

File: telethon/client/uploads.py [hunk 1]
```diff
@@ -423,7 +423,7 @@ async def _send_album(self: 'TelegramClient', entity, files, caption='',
                 ))
 
                 fm = utils.get_input_media(r.photo)
-            elif isinstance(fm, types.InputMediaUploadedDocument):
+            elif isinstance(fm, types.InputMediaUploadedDocument) or isinstance(fm, types.InputMediaDocumentExternal):
```
Comment (Lonami) [inline telethon/client/uploads.py:426]:
```suggestion
            elif isinstance(fm, (types.InputMediaUploadedDocument, types.InputMediaDocumentExternal)):
```

**Comment by kesha1225** [comment]
I want to do
```await client.send_file(entity="kesha1225", file=["link1.com/video.mp4", "link2.com/video.mp4"])```
but it isnt work and send only one video, this fix helps me with _send_album method. Maybe it not necessary. Just tell me how send list of urls as a album.

**Comment by Lonami** [comment]
What's the reason for closing this? Does your fix work or not?

**Comment by Lonami** [comment]
Passing `inputMediaPhotoExternal` to `messages.uploadMedia` results in `MEDIA_INVALID`, which makes me think links **cannot** be used in albums. Now there's a reason for this to be closed.

**Pull request closed without merging**